tests/probes/partition/test_probes_partition_offline_mode.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+. $builddir/tests/test_common.sh
+
+set -e -o pipefail
+
+function test_probes_partition {
+    probecheck "partition" || return 255
+
+    local ret_val=0;
+    local DF="${srcdir}/test_probes_partition_offline_mode.xml"
+    local RF="test_probes_partition_offline_mode.results.xml"
+    echo "result file: $RF"
+    local stderr=$(mktemp $1.err.XXXXXX)
+    echo "stderr file: $stderr"
+
+    [ -f $RF ] && rm -f $RF
+
+    tmpdir=$(mktemp -t -d "test_offline_mode_partition.XXXXXX")
+
+    ln -s /proc $tmpdir
```

Comment:
Does it make sense to parametrize this function and try it also on empty filesystem? just to verify if probe can handle it?

Reply:
Done, except for environmentvariable58, where it does not make any sense.